tasks/main.yml

```diff
@@ -11,11 +11,81 @@
     path: "{{ certbot_dir }}/certbot-auto"
     mode: 0755
 
+- name: Stop nginx
```

Comment:
Is it necessary to completely stop the webserver to generate the certs? I usually leave them running.